src/components/Avatar/Avatar.tsx

```diff
@@ -11,14 +11,14 @@ import Tooltip from "../Tooltip/Tooltip";
 import ClickableWrapper from "../Clickable/ClickableWrapper";
 import Dialog from "../Dialog/Dialog";
 import { iconSubComponentProps } from "../Icon/Icon";
-
-import "./Avatar.scss";
 import VibeComponentProps from "src/types/VibeComponentProps";
+import "./Avatar.scss";
 
 const AVATAR_CSS_BASE_CLASS = "monday-style-avatar";
 const bemHelper = BEMClass(AVATAR_CSS_BASE_CLASS);
 
 type BackgroundColors = typeof elementColorsNames[keyof typeof elementColorsNames];
+
```

Comment:
regarding tooltip props: There is tooltip pr which will be ready to review today. let's wait after it will merge to master and then you will be able to use TooltipProps interface for this